plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -210,24 +227,33 @@ static bool nvenc_update(void *data, obs_data_t *settings)
 	enc->height = enc->context->height;
 
 	info("settings:\n"
-	     "\trate_control: %s\n"
-	     "\tbitrate:      %d\n"
-	     "\tcqp:          %d\n"
-	     "\tkeyint:       %d\n"
-	     "\tpreset:       %s\n"
-	     "\tprofile:      %s\n"
-	     "\tlevel:        %s\n"
-	     "\twidth:        %d\n"
-	     "\theight:       %d\n"
-	     "\t2-pass:       %s\n"
-	     "\tb-frames:     %d\n"
-	     "\tGPU:          %d\n",
-	     rc, bitrate, cqp, enc->context->gop_size,
-	     preset, profile, level,
-	     enc->context->width, enc->context->height,
-	     twopass ? "true" : "false",
-	     enc->context->max_b_frames,
-	     gpu);
+		"\trate_control: %s\n"
```

Comment:
You changed the whitespace in this section.  Please revert your whitespace changes and present only the functional changes.